src/main/resources/application.conf

```diff
@@ -78,8 +78,8 @@ bi-dataload {
 
     links {
       dir = ${?BI_DATALOAD_LINKS_DATA_DIR}
-      parquet = "legal_units.parquet"
-      parquet = ${?BI_DATALOAD_LINKS_DATA_PARQUET}
+      linksFile = "legal_units.parquet"
+      linksFile = ${?BI_DATALOAD_LINKS_DATA_FILE}
```

Comment:
the `links` in `linksFile` is superfluous as its already contained in the `links` block.